src/libraries/System.Memory/src/System/Buffers/SequenceReader.Search.cs

```diff
@@ -405,6 +405,43 @@ private bool TryReadToAnyInternal(out ReadOnlySequence<T> sequence, ReadOnlySpan
             return false;
         }
 
+        /// <summary>
+        /// Try to read everything up to the given <paramref name="delimiter"/>.
+        /// </summary>
+        /// <param name="span">The read data, if any.</param>
+        /// <param name="delimiter">The delimiter to look for.</param>
+        /// <param name="advancePastDelimiter">True to move past the <paramref name="delimiter"/> if found.</param>
+        /// <returns>True if the <paramref name="delimiter"/> was found.</returns>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Is this really needed?

Reply:
Hmm.. thought I was copying the other `TryReadTo` methods, but they don't seem to have it.

Removed